forge-script/PermissionlessPoolWithdrawAdminFeeCheck.s.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "forge-std/Script.sol";
+import "@openzeppelin/contracts-4.4.0/token/ERC20/utils/SafeERC20.sol";
+
+interface IPermissionlessSwap {
+    function withdrawAdminFees() external payable;
+
+    function owner() external view returns (address);
+
+    function feeCollector() external view returns (address);
+
+    function getVirtualPrice() external view returns (uint256);
+}
+
+contract PermissionlessPoolWithdrawAdminFeeCheckScript is Script {
+    uint256 snapshot;
+    IPermissionlessSwap basePool =
+        IPermissionlessSwap(
+            address(0x1933B1D4A372Ecbc3713D92F875b330846ae6A2b)
```

Comment:
is it possible to reference constants, either from a file or hre?

Reply:
Yeah need to implement a custom function that can read from deployment json. PR incoming for this.